.github/workflows/release.yml

```diff
@@ -153,7 +155,8 @@ jobs:
       matrix:
         # This job intermittently fails on EKS runners and must be run on standalone until
         # https://www.pivotaltracker.com/story/show/185045683 is resolved.
-        os: [[self-hosted, Linux, X64, build]]
+        # os: [[self-hosted, Linux, X64, testing, v2], [self-hosted, Linux, ARM64,  testing, v2]]
+        os: [[self-hosted, Linux, X64, testing, v2]]
```

Comment:
What is the purpose of "testing" and "v2"?

Reply:
These are the new eks runners.